.github/workflows/metadata-check.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler/blob/master/README.md
+
+name: Metadata Checker 
+on:
+  push:
+    branches: 
+      - dev
+      - master
+
+concurrency:
+  group: metadata-checker-${{github.ref}}
+  cancel-in-progress: true
+
+jobs:
+  metadata-checker:
+    runs-on: ubuntu-latest
+    if: (github.repository == 'AzureAD/microsoft-authentication-library-for-js')
```

Comment:
This line isn't needed

Reply:
I have removed this line from the workflow file